superflore/generators/bitbake/run.py

```diff
@@ -81,9 +81,8 @@ def main():
 
     # clone current repo
     selected_targets = active_distros
-    # try:
-    link_existing_files(args.ros_distro)
-    """
+    try:
+        link_existing_files(args.ros_distro)
```

Comment:
This looks to be linking args.ros_distro, but the exception removes all active distros?